defaults/main.yml

```diff
@@ -1456,6 +1456,26 @@ owncloud__nginx_access_log_assets: True
 # TODO: Confirm that this variable does what it says.
 owncloud__nginx_client_body_temp_path: ''
 
+owncloud__nginx__php_status: False
+owncloud__nginx__php_status_location:
+  pattern: '~ ^/(php_status|php_ping)$'
+  options: |
+    access_log off;
+    {% if nginx_status_localhost %}
+    {% for address in nginx_status_localhost | unique %}
+    allow {{ address }};
+    {% endfor %}
+    {% endif %}
+    {% if nginx_status %}
+    {% for address in nginx_status | unique %}
+     allow {{ address }};
+    {% endfor %}
+    {% endif %}
+    deny all;
+    include fastcgi_params;
+    fastcgi_index index.php;
+    fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+    fastcgi_pass php_owncloud;
```

Comment:
There is a comment about use "php" server configuration:

## Not used so that the exact order of locations from the upstream nginx example can be used.
    # type: 'php'
    # php_upstream: 'php_owncloud'
    # php_limit_except: False
    # php5_location: '^/(?:index|remote|public|cron|core/ajax/update|status|ocs/v[12]|updater/.+|ocs-provider/.+|core/templates/40[34])\.php(?:$|/)'
    # php_options: |
#   fastcgi_read_timeout {{ owncloud__timeout }};

The problem for use "php" server configuration seems order of locations.